src/schema/base.yaml

```
@@ -297,6 +266,24 @@ classes:
           a percentage.
         required: false
 
+  AliasMixin:
+    mixin: true
+    abstract: true
```

```suggestion
    mixin: true
```

I think specifying mixin true is probably enough.